go.mod

```diff
@@ -0,0 +1,118 @@
+module github.com/openshift/installer
+
+go 1.12
+
+require (
+	cloud.google.com/go/bigtable v1.1.0 // indirect
+	github.com/Azure/azure-sdk-for-go v36.2.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.9.2
+	github.com/Azure/go-autorest/autorest/adal v0.8.1-0.20191028180845-3492b2aff503
+	github.com/Azure/go-autorest/autorest/azure/auth v0.4.0
+	github.com/Azure/go-autorest/autorest/to v0.3.0
+	github.com/Netflix/go-expect v0.0.0-20180615182759-c93bf25de8e8
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/awalterschulze/gographviz v0.0.0-20170410065617-c84395e536e1
+	github.com/aws/aws-sdk-go v1.25.3
+	github.com/containers/image v3.0.2+incompatible
+	github.com/coreos/ignition v0.28.0
+	github.com/dmacvicar/terraform-provider-libvirt v0.6.0
+	github.com/dustinkirkland/golang-petname v0.0.0-20191129215211-8e5a1ed0cff0 // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/golang/mock v1.3.1
+	github.com/gophercloud/gophercloud v0.6.1-0.20191019020556-0907b320e0ac
+	github.com/gophercloud/utils v0.0.0-20191020172814-bd86af96d544
+	github.com/hashicorp/go-azure-helpers v0.10.0
+	github.com/hashicorp/go-plugin v1.0.1
+	github.com/hashicorp/logutils v1.0.0
+	github.com/hashicorp/terraform v0.12.17
+	github.com/hashicorp/terraform-plugin-sdk v1.3.0
+	github.com/hinshun/vt10x v0.0.0-20180616224451-1954e6464174
+	github.com/kr/fs v0.1.0 // indirect
+	github.com/libvirt/libvirt-go v5.0.0+incompatible
+	github.com/metal3-io/baremetal-operator v0.0.0
+	github.com/metal3-io/cluster-api-provider-baremetal v0.0.0
+	github.com/mitchellh/cli v1.0.0
+	github.com/onsi/ginkgo v1.10.3
+	github.com/onsi/gomega v1.7.1
+	github.com/openshift-metal3/terraform-provider-ironic v0.1.8
+	github.com/openshift/api v3.9.1-0.20191014195513-c9253efc14f4+incompatible
+	github.com/openshift/client-go v0.0.0-20191001081553-3b0e988f8cb0
+	github.com/openshift/cloud-credential-operator v0.0.0-20190905120421-44ed18ef8496
+	github.com/openshift/cluster-api v0.0.0-20190923092624-4024de4fa64d
+	github.com/openshift/cluster-api-provider-gcp v0.0.0-20190826205919-0cd5daa07e0d
+	github.com/openshift/cluster-api-provider-libvirt v0.0.0-20190613141010-ecea5317a4ab
+	github.com/openshift/library-go v0.0.0-20191003152030-97c62d8a2901
+	github.com/openshift/machine-config-operator v0.0.0-20191029224459-e8050812d506
+	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
+	github.com/pkg/sftp v1.10.0
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
+	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	github.com/stoewer/go-strcase v1.1.0 // indirect
+	github.com/stretchr/testify v1.4.0
+	github.com/terraform-providers/terraform-provider-aws v0.0.0-00010101000000-000000000000
```

Comment:
it would be great if you can provide some context around this _off_ import

Reply:
the v0.0.0... are inserted automatically for all packages that are replaced/pinned to a specific version.
I can note it somewhere.

Reply:
I'll drop this once we can update that lib (requires Go 1.13 first), see also https://github.com/openshift/installer/pull/2676